alf/utils/distributions.py

```diff
@@ -201,7 +201,11 @@ def upper_bound(self):
     @property
     def mode(self):
         """Mode of this distribution."""
-        return self._loc.clamp(self._lower_bound, self._upper_bound)
+        result = torch.where(self._loc < self._lower_bound, self._lower_bound,
```

Comment:
using torch.maximum() and minimum() will be better

Reply:
Definitely. Fixed.